src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -39,37 +38,23 @@
     public void visitDashboard() {
         testPatient = createTestPatient();
         initiateWithLogin();
-        new TestData.TestVisit(testPatient.uuid, TestData.getAVisitType(),
```

Comment:
Are we not using this method intatiation any where ?

Reply:
yes i decided to change direction and use the **find record link** instead of the visits link. i was hoping to get rid of the [snag here](https://reports.cucumber.io/reports/190a1576-922e-4959-ba58-eb92d3b1fa44) but unfortunately it doesn't.

please @sherrif10 look through and maybe you could tell me why the hook (@after) or `destroy()` method fail